suite/tests/CMakeLists.txt

```diff
@@ -3754,7 +3754,7 @@ endforeach(run)
 if (NOT DEBUG AND UNIX AND X64 AND NOT CMAKE_COMPILER_IS_CLANG)
   set_tests_properties(
     samples_proj
-    code_api|common.eflags
+    code_api|common.${control_flags}
```

Comment:
LGTM but best to first test locally in a cross-compile release build for A64 (are all of the tests below on A64?  If not best to split the list (or (longer-term?) move to setting these on individual tests so we don't have this separate control point to keep in sync).

Reply:
I don't have any A64 machine. @AssadHashmi , could you try out this patch?

Reply:
@hgreving2304 no need to run, just build: you should be able to use the same cross-compiler packages used by Travis.

Reply:
True. Ok, done, works.

Reply:
Or, just temporarily enable BUILD_TESTS for the arm-release-external-64 build on Travis.

Reply:
(Though for the future it is useful to have a local cross-compile build dir so I would suggest setting it up regardless.)